### PR TITLE
chore: improve beets maintenance path

### PR DESCRIPTION
**Pull request by lphuc2250gma**
Summary:
- Add or tighten focused edge-case tests or type assertions in test/test_util.py, test/test_files.py related to Python typing, tests, CLI ergonomics, observability; avoid docs-only changes and broad refactors.
- Keep the change narrow so it is straightforward to review.

Notes:
- I kept this scoped to the relevant implementation and tests.

### CODE REVIEW

File: test/test_files.py [hunk 1]
```diff
@@ -245,6 +245,12 @@ def test_components_works_on_relative(self):
         a = ["a", "b", "c"]
         assert util.components(p) == a
 
+    def test_ancestry_works_on_empty_string(self):
```
Comment (snejus) [inline test/test_files.py:248]: Would you mind refactoring these tests to use `pytest.mark.parametrize`?

File: uv.lock [hunk 2]
```diff
@@ -0,0 +1,3 @@
+version = 1
```
Comment (snejus) [inline uv.lock:1]: What is this one for?
